### PR TITLE
ged / import user groups

### PR DESCRIPTION
**Pull request by pvilalta**
https://platform-dev-import-user-groups.bimdata.io

## Description

One point that I would like to discuss
With BIMDataMenu component, if 2 items has the same name and has children, when one is over both display their children.
In my case, if 2 projects has the same name, this limitation will occurs.

any thought ? 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have tested my code.

### CODE REVIEW

File: src/components/specific/groups/group-import/GroupImport.vue [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+              ...project,
+              text: project.name,
+              children
+            };
+          })
+      );
+      open();
+    };
+
+    const importGroup = async children => {
```
Comment (NicolasRichel) [inline src/components/specific/groups/group-import/GroupImport.vue]: This function looks like something that can be "lifted up" into the state directly.
Let's have a talk about that when you have a moment. ;)

**Review by NicolasRichel — CHANGES_REQUESTED** [review]

**Review by NicolasRichel — APPROVED** [review]

**Review by NicolasRichel — APPROVED** [review]

**Review by NicolasRichel — APPROVED** [review]

File: src/components/specific/groups/group-import/GroupImport.vue [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            </div>
+            <div class="group-import__children__menu__header__divider" />
+          </div>
+        </template>
+        <template #child-item="{ child }">
+          <BIMDataCheckbox
+            class="group-import__children__menu__item--checkbox"
+            margin="0 6px 0 0"
+            :modelValue="
+              Boolean(
```
Comment (Kurtil) [inline src/components/specific/groups/group-import/GroupImport.vue:91]: `Boolean(arr.find(func))` => `arr.some(func)`

Comment (Kurtil) [reply src/components/specific/groups/group-import/GroupImport.vue:91]: In this case the `Boolean()` call may be still used to parse undefined into false.

**Review by Kurtil — CHANGES_REQUESTED** [review]

**Review by Kurtil — APPROVED** [review]

**Review by GabBimdata — APPROVED** [review]

**Review by NicolasRichel — APPROVED** [review]

**Pull request merged**